faster_raster.go

```diff
@@ -352,7 +352,16 @@ func (r *Rasterizer) processOne(req *RasterRequest) {
 	// Create a clone of r.Ctx for objects that are allocated for rendering the current page
 	ctx := C.fz_clone_context(r.Ctx)
 	if ctx == nil {
-		req.ReplyChan <- &RasterReply{Error: errors.New("failed to clone context")}
+		select {
+		case req.ReplyChan <- &RasterReply{Error: errors.New("failed to clone context")}:
+			log.Warnln("Failed to clone context!")
```

Comment:
If you're going to reply with the error, we should probably not also log it.

Reply:
Thanks! I've seen the same done [just above](https://github.com/Nitro/lazypdf/blob/243fa9e09712d0409848fe89b1eb478f8e462fc3/faster_raster.go#L342) already, so decided to follow the same pattern. I'll try to refactor this into a function in a future PR, since we use the same pattern in multiple places.

Reply:
Well, the subtle difference there is that the instance you cite needs state that is not available in the error. That error returns a named error and so can't modify the text to include debugging information. Unless the consumer of that reply is checking error identity, though, that could be replaced with an `errors.New()` call also

Reply:
Indeed, good point. We're using that [here](https://github.com/Nitro/lazypdf/blob/243fa9e09712d0409848fe89b1eb478f8e462fc3/faster_raster.go#L41). OK, I'll drop this log for now and see if I want to improve it in a future refactor.